SnipperAPI/Controllers/SnippetsControllers.cs

```diff
@@ -57,9 +93,39 @@ public ActionResult<IEnumerable<Snippet>> GetAllSnippets()
             }
 
             return Ok(Snippets);
+        }*/
+
+
+        [HttpGet("snippets")]
+        [Authorize] // Requires authentication
+        public ActionResult<IEnumerable<Snippet>> GetAllSnippets()
+        {
+            // Get the current user
+            var currentUser = _userManager.GetUserAsync(User).Result;
```

Comment:
Similar issue here but by fixing either this or the other comment around this will resolve them both